pyracing/constants.py

```diff
@@ -34,6 +35,9 @@
 URL_TOTALREGISTERED = (mSite + '/GetTotalSessionJoinedCountsBySeason')
 URL_RACEGUIDE = (mSite + '/GetRaceGuide')
 URL_ACTIVEOP_COUNT = (mSite + '/GetActiveOpenPracticeCount')
+# can be used without parameters for all hosted sessions
+URL_HOSTED_SESSIONS = (mSite + '/GetHostedSessions'
+                       'ts=0' or 'privateSessionID=1801560')
```

Comment:
this `or` does nothing. it will check if the first value is `truthy` which it is, so it will never hit the 2nd part of this `or`